projectmanagement/src/main/java/life/qbic/projectmanagement/domain/project/sample/Sample.java

```diff
@@ -121,4 +122,21 @@ public BiologicalReplicateId biologicalReplicateId() {
     return this.biologicalReplicateId;
   }
 
+  public AnalysisMethod analysisMethod() {
+    return this.analysisMethod;
+  }
+
+  static class AnalysisMethodConverter implements AttributeConverter<AnalysisMethod, String> {
+    
+    @Override
+    public String convertToDatabaseColumn(AnalysisMethod analysisMethod) {
+      return analysisMethod.name();
+    }
+
+    @Override
+    public AnalysisMethod convertToEntityAttribute(String s) {
+      return AnalysisMethod.valueOf(s);
+    }
```

Comment:
This leads to an exception if no analysis value is present in the database => not backwards compatible with existing experiments/samples

Reply:
since we have no official production release, I think backwards compatibility is not needed to be considered. The domain expects an entry, everything else is not accepted.